src/main/webapp/index.html

```diff
@@ -44,37 +46,62 @@ <h2 class="panel-row__header u-align-with-first-panel u-text-shadow">Services (a
             </div>
-            <div class="panel__content-entry">
-              <img class="panel__content-icon" alt="clock-icon" src="assets/clock.png">
-              <p class="panel__text-large"><span id="gmailUnreadEmailsHours"></span> from the last <span id="gmailMHours"></span> hours
+            <p class="panel__text-data-point">Unread Emails:
+          </div>
+          <div class="panel__content-entry">
+            <img class="panel__content-icon" alt="clock-icon" src="assets/clock.png">
+            <p class="panel__text-large"><span id="gmail-unread-emails-hours"></span> from the last <span id="gmail-m-hours"></span> hours
+          </div>
+          <div class="panel__content-entry">
+            <img class="panel__content-icon" alt="flag-icon" src="assets/flag.png">
+            <p class="panel__text-large"><span id="gmail-unread-important-emails"></span> marked important
+          </div>
+          <div class="panel__content-entry">
+            <div class="panel__round-letter">
+              <p class="panel__text-data-point u-text-white" id="gmail-sender-initial">
             </div>
-            <div class="panel__content-entry">
-              <img class="panel__content-icon" alt="flag-icon" src="assets/flag.png">
-              <p class="panel__text-large"><span id="gmailUnreadImportantEmails"></span> marked important
+            <p class="panel__text-large"><span id="gmail-sender"></span> sent the most unread emails
+          </div>
+        </div>
+        <!-- Settings section -->
+        <h3 hidden class="panel__section-heading u-remove-top-margin gmail-settings">Settings:</h3>
+        <div hidden class="panel__text-body gmail-settings">
+          <div class="panel__content-entry u-remove-top-margin">
+            <p class="panel__text-large">Search <span class="u-bold-underline" id="gmail-settings-n-days">7</span> days of email
+            <div class="panel__two-buttons-incremental u-right-align">
+              <a class="panel__button-incremental" onclick="incrementElement('gmail-settings-n-days')"><span>+</span></a>
+              <a class="panel__button-incremental" onclick="decrementElement('gmail-settings-n-days')"><span>-</span></a>
             </div>
-            <div class="panel__content-entry">
-              <div class="panel__round-letter">
-                <p class="panel__text-data-point u-text-white" id="gmailSenderInitial">
-              </div>
-              <p class="panel__text-large"><span id="gmailSender"></span> sent the most unread emails
+          </div>
+          <div class="panel__content-entry">
+            <p class="panel__text-large">Search <span class="u-bold-underline" id="gmail-settings-m-hours">3</span> hours of recent email
+            <div class="panel__two-buttons-incremental u-right-align">
+              <a class="panel__button-incremental" onclick="incrementElement('gmail-settings-m-hours')"><span>+</span></a>
```

Comment:
Does the increment have a limit? Would it make sense to add an extra day when it reaches 24?

Reply:
Yes the limit is 30 - and I actually think maybe the limit on this "hours" property should be 23. The whole point is to see how many recent emails there are, and by allowing "30" hours it's defeating the purpose (plus that's an awkward amount of time). Fixing

Reply:
Sgtm